colbert/infra/config/settings.py

```diff
@@ -30,6 +30,8 @@ class RunSettings:
     total_visible_gpus = torch.cuda.device_count()
     gpus: int = DefaultVal(total_visible_gpus)
 
+    use_rank1_fork: bool = DefaultVal(False)
```

Comment:
Just a quick note (well, two):

- Should this not be the opposite logic? e.g. defaults to `True` (and the associated logic changes in the code)? I feel like it'd make more sense with the naming
- Do we want this to be a `config` item rather than a flag passed to the function? I think either works fine and maybe having it as part of config is more fitting for how the rest of the code is structured.

Reply:
I was using fork in the sense of a code fork but will revert to fork as in multiprocessing! thanks for explaining the terminology!

Reply:
Thanks, all looking good to me now! Just needs Omar's final approval 👍